cogs/stackoverflow.py

```diff
@@ -0,0 +1,164 @@
+import re
+import asyncio
+import aiohttp
+import logging
+from urllib.parse import quote
+from typing import List
+from html.parser import HTMLParser
+
+# Discord.py Library
+from discord import (
+    Embed, 
+    Colour,
+    Reaction,
+    Member
+)
+from discord.ext import commands
+
+# CONSTANTS
+PAGINATION_EMOJIS = ('⏪', '◀️', '⏹️', '▶️', '⏩')
+log = logging.getLogger(__name__)
+
+class Stackoverflow(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.session = aiohttp.ClientSession()
+
+    async def stack_request(self, ctx, title: str) -> List[dict]:
+        """-------------- STACKOVERFLOW REQUEST --------------"""
+        title = quote(title)
+        filtered_request = {}
+        api = f"https://api.stackexchange.com/2.2/search/advanced?key=U4DMV*8nvpm3EOpvf69Rxw((&site=stackoverflow&order=desc&sort=votes&closed=False&title={title}&filter=default"
```

Comment:
Added the key used on the stackoverflow api (got from ctrl+shift+i), so that we can have more results per day :D